### PR TITLE
fix v1

### PR DESCRIPTION
**Pull request by gc-fu**

### CODE REVIEW

File: csrc/xpu/attention_xpu.cpp [hunk 1]
```diff
@@ -144,6 +144,336 @@ inline float block_sum(
...
+            const scalar_t* key_head =
+                (const scalar_t*)key +
+                physical_block_number * k_cache_stride_tokens +
+                kv_head_idx * k_cache_stride_head +
+                which_slot * k_cache_stride_block_size;
+            simd<scalar_t, HD> key_row = block_load<scalar_t, HD>(key_head);
+            slm_block_store(key_slm_offset + tid * HD * sizeof(scalar_t),
+                            key_row);
+
+            // [num_blocks, num_kv_heads, head_size, block_size]
```
Comment (xiangyuT) [inline csrc/xpu/attention_xpu.cpp:425]: Is this correct?

Comment (gc-fu) [reply csrc/xpu/attention_xpu.cpp:425]: Will remove, after reshape this is not correct...

File: vllm/attention/backends/ipex_attn.py [hunk 2]
```diff
@@ -463,8 +463,13 @@ def forward(
                     value = value.repeat_interleave(self.num_queries_per_kv,
                                                     dim=1)
                 import vllm._C.ops
+                assert self.head_size == 128 or self.head_size == 64
                 value = os.environ.get('USE_CONTEXT_V1')
-                if self.head_size == 128 and value is None:
+                if using_gqa_kernel:
```
Comment (xiangyuT) [inline vllm/attention/backends/ipex_attn.py:468]: Maybe we could rename `using_gqa_kernel` to `using_ipexllm_kvcache`?

Comment (gc-fu) [reply vllm/attention/backends/ipex_attn.py:468]: Make sense, will change

**Review by xiangyuT — APPROVED** [review]
LGTM

**Pull request merged**